node/runtime/pangolin/src/pallets/bridge_messages.rs

```diff
@@ -1,58 +1,47 @@
 pub use pallet_bridge_messages::Instance1 as WithPangoroMessages;
 
-// --- paritytech ---
-use bp_messages::MessageNonce;
-use bp_pangolin::{
-	AccountIdConverter, MAX_SINGLE_MESSAGE_DELIVERY_CONFIRMATION_TX_WEIGHT,
-	MAX_UNCONFIRMED_MESSAGES_IN_CONFIRMATION_TX, MAX_UNREWARDED_RELAYERS_IN_CONFIRMATION_TX,
-};
-use bp_runtime::PANGORO_CHAIN_ID;
-use pallet_bridge_messages::Config;
 // --- darwinia-network ---
-use crate::{
-	pangoro_messages::{
-		FromPangoroMessageDispatch, FromPangoroMessagePayload, PangolinToPangoroMessagesParameter,
-		Pangoro, ToPangoroMessagePayload, ToPangoroMessageVerifier,
-	},
-	*,
-};
+use crate::{bridges_message::bm_pangoro, *};
+use bp_messages::MessageNonce;
+use bp_runtime::{ChainId, PANGORO_CHAIN_ID};
 use darwinia_fee_market::s2s::{
 	FeeMarketMessageAcceptedHandler, FeeMarketMessageConfirmedHandler, FeeMarketPayment,
 };
 use darwinia_support::evm::{ConcatConverter, IntoAccountId, IntoH160};
+use pallet_bridge_messages::Config;
 
 frame_support::parameter_types! {
 	pub const MaxMessagesToPruneAtOnce: MessageNonce = 8;
 	pub const MaxUnrewardedRelayerEntriesAtInboundLane: MessageNonce =
-		MAX_UNREWARDED_RELAYERS_IN_CONFIRMATION_TX;
+		bp_pangoro::MAX_UNREWARDED_RELAYERS_IN_CONFIRMATION_TX;
```

Comment:
This should be `bp_panolin`?

Reply:
https://github.com/darwinia-network/darwinia-bridges-substrate/blob/main/bin/rialto/runtime/src/lib.rs#L423